lib/crypto/butil.js

```diff
@@ -35,7 +35,7 @@ module.exports.xorBuffers = function xorBuffers(buffer1, buffer2) {
 module.exports.unbuffer = function unbuffer(object, inplace) {
   var keys = Object.keys(object)
   var copy = inplace ? object : {}
-  for (var i = 0; i < keys.length; i++) {
+  for (var i = 0; i < keys.length; i ++) {
```

Comment:
Following Firefox desktop style, my preferences are:

1) `i++` and `++i`
2) `if (! thing)`